### DISCUSSION TITLE
Regarding not able to update type in group field in ACF in Flynt

### DISCUSSION THREAD

**Discussion by rajpatel2435** [flyntwp/flynt#559] [General]
Hey @timohubois @aaronmeder 

When I updated the field inside ACF Pro lugin inside flynt it is not updating at all.

![image](https://github.com/flyntwp/flynt/assets/81604941/c5d2ddae-b55f-4d66-86ca-4fcf35d7236e)

does anyone has solution?

Thanks

**Comment by aaronmeder** [comment]
@rajpatel2435 please apply the changes made in this pull request:
https://github.com/flyntwp/flynt/pull/545

This was very recently merged into the main branch and should fix the issue.

**Comment by rajpatel2435** [comment]
Hey @aaronmeder 

Thank you so much.

It works well.

Thanks
